### PR TITLE
update-signal-workflows-description

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

File: Controls/resources/catalog/Check_For_Signals.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
         xmlns="urn:proactive:jobdescriptor:3.13" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd"  name="Check_For_Signals" projectName="4. Signal Templates" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
   <description>
-    <![CDATA[ A template workflow that sends a ready notification for all the signals specified in the variable SIGNALS, then loops until one signal among those specified is received by the job. ]]>
+    <![CDATA[ A template workflow composed by two tasks. The first sends a ready notification for all of the signals specified in the variable SIGNALS. The second task checks if one signal is received, and if not, the workflow iterates in a loop where each iteration lasts one minute. This workflow allows users to do an iterative processing, and decide at each iteration to continue this processing or not, i.e., (i) declare being ready to receive one or more signals, (ii) do some processing, then (iii) receive signals and decide whether to continue the processing or not. ]]>
```
Comment (mboussaa) [inline Controls/resources/catalog/Check_For_Signals.xml:6]: for all specified signals in the 'SIGNALS' variable

Comment (mboussaa) [reply Controls/resources/catalog/Check_For_Signals.xml:6]: iterates in a loop where each iteration lasts one minute 
=>
the workflow runs in a loop with one-minute iteration duration

File: Controls/resources/catalog/Wait_For_Signals.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xmlns="urn:proactive:jobdescriptor:3.13" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd"  name="Wait_For_Signals" projectName="4. Signal Templates" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
   <description>
-    <![CDATA[ A template workflow that sends a ready notification for all the signals specified in the variable SIGNALS, then loops until one signal among those specified is received by the job. ]]>
+    <![CDATA[ A template workflow that contains a single task, which sends a ready notification for the signals specified in the variable SIGNALS, then checks if one or more signals are received by the job. If not, the workflow iterates in a loop where each iteration lasts one minute. This workflow allows users to iteratively wait for the reception of one or more signals, then trigger some processing. ]]>
```
Comment (mboussaa) [inline Controls/resources/catalog/Wait_For_Signals.xml:6]: same

File: Controls/resources/catalog/Wait_For_Signals_Reactive.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xmlns="urn:proactive:jobdescriptor:3.13" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd"  name="Wait_For_Signals_Reactive" projectName="4. Signal Templates" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
   <description>
-    <![CDATA[ A template workflow that sends a ready notification for all the signals specified in the variable SIGNALS, then waits (i.e., performs a blocking wait) until one signal among those specified is received by the job. ]]>
+    <![CDATA[ A template workflow that sends a ready notification for all the signals specified in the variable SIGNALS, then waits until one signal among those specified is received. This workflow contains no loop. It performs a blocking wait, then immediately triggers some processing at the reception of a signal. ]]>
```
Comment (mboussaa) [inline Controls/resources/catalog/Wait_For_Signals_Reactive.xml:6]: same and signal reception

File: Controls/resources/catalog/wait_for_signal_with_variables.xml [hunk 4]
```diff
@@ -3,7 +3,7 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xmlns="urn:proactive:jobdescriptor:3.13" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd"  name="Wait_For_Signal_With_Variables" projectName="4. Signal Templates" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
   <description>
-    <![CDATA[ A template workflow that waits until one signal (with input parameters) that is specified in the variable SIGNAL is added to the set of job signals. ]]>
+    <![CDATA[ A template workflow that deals signals having input parameters. It contains a single task that sends a ready notification for all the signals (with input parameters) specified in the variable SIGNALS, then waits until one signal (among those specified) is received. The signal's input parameters have predefined values that can be changed when sending the signal. ]]>
```
Comment (mboussaa) [inline Controls/resources/catalog/wait_for_signal_with_variables.xml:6]: that deals signals => this is ugly, could we have another description

Comment (nebilbenmabrouk) [reply Controls/resources/catalog/wait_for_signal_with_variables.xml:6]: deals **with** signals

Comment (oanaBiancaSchiopu) [reply Controls/resources/catalog/wait_for_signal_with_variables.xml:6]: that deals signals => that handles signals?

File: Controls/resources/catalog/Check_For_Signals.xml (hunk 1 above)
Comment (nebilbenmabrouk) [reply Controls/resources/catalog/Check_For_Signals.xml:6]: - 'A template workflow composed **of** two tasks. The first sends a ready notification for **all the** signals specified in the variable SIGNALS'
- 'the workflow runs in a loop where each iteration lasts one minute'

File: Controls/resources/catalog/wait_for_signal_with_variables.xml (hunk 4 above)
Comment (oanaBiancaSchiopu) [reply Controls/resources/catalog/wait_for_signal_with_variables.xml:6]: signal's => remove 's . Why should 's be removed?

File: Controls/resources/catalog/Wait_For_Signals_Reactive.xml (hunk 3 above)
Comment (nebilbenmabrouk) [reply Controls/resources/catalog/Wait_For_Signals_Reactive.xml:6]: - 'at any signal reception'

File: Controls/resources/catalog/wait_for_signal_with_variables.xml (hunk 4 above)
Comment (nebilbenmabrouk) [reply Controls/resources/catalog/wait_for_signal_with_variables.xml:6]: > that deals signals => that handles signals?
We can also use: 'A template workflow that uses signals having input parameters'

Comment (nebilbenmabrouk) [reply Controls/resources/catalog/wait_for_signal_with_variables.xml:6]: > signal's => remove 's . Why should 's be removed?

yes, we can remove the 's. **The signal input parameters**

Comment (mboussaa) [reply Controls/resources/catalog/wait_for_signal_with_variables.xml:6]: we don’t usually use the possessive 's with things like signals. Then, it makes the sentence easier to read

File: Controls/resources/catalog/Check_For_Signals.xml (hunk 1 above)
Comment (oanaBiancaSchiopu) [reply Controls/resources/catalog/Check_For_Signals.xml]: @mboussaa @nebilbenmabrouk  which one should I commit? :D

**Comment by oanaBiancaSchiopu** [comment]
retest this please

**Review by nebilbenmabrouk — APPROVED** [review]

**Review by nebilbenmabrouk — APPROVED** [review]

**Review by mboussaa — APPROVED** [review]

**Pull request merged**